assets/sass/05-blocks/quote/_style.scss

```diff
@@ -16,18 +16,40 @@
 		}
 	}
 
+	&::before {
+		color: currentColor;
+		content: "\201C";
+		display: block;
+		font-size: 3rem;
+		font-weight: 700;
+		line-height: 1;
+		position: absolute;
+		top: 4px;
+		left: -20px;
+	}
+
 	p {
+		color: var(--global--color-primary);
```

Comment:
Why is this needed?

Reply:
Oh, this is also inherited. Removing it.